File: RxKit/src/main/java/com/tamsiree/rxkit/interfaces/OnRxCamera.java
package com.tamsiree.rxkit.interfaces;

import java.io.File;

/**
 *
 * @author Tamsiree
 * @date 2017/8/9
 */

public interface OnRxCamera {

    void onBefore();

    void onSuccessCompress(File filePhoto);

    void onSuccessExif(File filePhoto);
}

File: RxKit/src/main/java/com/tamsiree/rxkit/interfaces/OnRxCamera.kt
package com.tamsiree.rxkit.interfaces

import java.io.File

/**
 *
 * @author Tamsiree
 * @date 2017/8/9
 */
interface OnRxCamera {
    fun onBefore()
    fun onSuccessCompress(filePhoto: File?)
    fun onSuccessExif(filePhoto: File?)
}